moveit_ros/planning/planning_pipeline/include/moveit/planning_pipeline/planning_pipeline.h

```diff
@@ -63,102 +63,71 @@ class MOVEIT_PLANNING_PIPELINE_EXPORT PlanningPipeline
 public:
   /** \brief When motion plans are computed and they are supposed to be automatically displayed, they are sent to this
    * topic (moveit_msgs::msg::DisplauTrajectory) */
-  static const std::string DISPLAY_PATH_TOPIC;
-
+  inline static const std::string DISPLAY_PATH_TOPIC = std::string("display_planned_path");
   /** \brief When motion planning requests are received and they are supposed to be automatically published, they are
    * sent to this topic (moveit_msgs::msg::MotionPlanRequest) */
-  static const std::string MOTION_PLAN_REQUEST_TOPIC;
-
+  inline static const std::string MOTION_PLAN_REQUEST_TOPIC = std::string("motion_plan_request");
   /** \brief When contacts are found in the solution path reported by a planner, they can be published as markers on
    * this topic (visualization_msgs::MarkerArray) */
-  static const std::string MOTION_CONTACTS_TOPIC;
+  inline static const std::string MOTION_CONTACTS_TOPIC = std::string("display_contacts");
 
   /** \brief Given a robot model (\e model), a node handle (\e pipeline_nh), initialize the planning pipeline.
       \param model The robot model for which this pipeline is initialized.
       \param node The ROS node that should be used for reading parameters needed for configuration
-      \param the parameter namespace where the planner configurations are stored
-      \param planning_plugin_param_name The name of the ROS parameter under which the name of the planning plugin is
-     specified
-      \param adapter_plugins_param_name The name of the ROS parameter under which the names of the request adapter
-     plugins are specified (plugin names separated by space; order matters)
+      \param parameter_namespace parameter namespace where the planner configurations are stored
   */
   PlanningPipeline(const moveit::core::RobotModelConstPtr& model, const std::shared_ptr<rclcpp::Node>& node,
-                   const std::string& parameter_namespace,
-                   const std::string& planning_plugin_param_name = "planning_plugin",
-                   const std::string& adapter_plugins_param_name = "request_adapters");
+                   const std::string& parameter_namespace);
```

Comment:
I think the one comment I have is on this interface.

By removing the extra 2 string arguments, we're breaking compatibility. Is there path to maybe not removing this interface, but marking it with a deprecation warning, at least?

Reply:
I don't think that this has a huge impact and I think that very few people (if any) use the pure PlanningPipeline class and I'd be surprised if they manually changed the parameter name of the planning_plugin and request_adapters. The main reason for removing this interface was that I felt it was relatively pointless, so I did not bother about keeping a deprecated version. If you'd prefer a proper deprecation I can add that :+1:

Reply:
Nah, I believe you if you say it's not gonna affect a ton of people. Let's do this!